### DISCUSSION TITLE
How to use the consent plugin on Ionic

### DISCUSSION THREAD

**Discussion by felipe-ff** [admob-plus/admob-plus#287] [Q&A]
On the official website, there is a "Request Consent" topic, but only for the cordova version.

When I try to install it and try to use it on ionic I get the following error: "Cannot find name 'consent'.ts(2304)".

How to use the consent plugin on ionic projects?

**Comment by EMI-INDO** [comment]
> On the official website, there is a "Request Consent" topic, but only for the cordova version.
> 
> When I try to install it and try to use it on ionic I get the following error: "Cannot find name 'consent'.ts(2304)".
> 
> How to use the consent plugin on ionic projects?

@felipe-ff 

### Have you tried this code

https://github.com/admob-plus/admob-plus/blob/master/examples/cordova-consent/www/js/index.js


### If the application has been released

https://support.google.com/fundingchoices/answer/9180084

**Comment by felipe-ff** [comment]
> > On the official website, there is a "Request Consent" topic, but only for the cordova version.
> > When I try to install it and try to use it on ionic I get the following error: "Cannot find name 'consent'.ts(2304)".
> > How to use the consent plugin on ionic projects?
> 
> @felipe-ff
> 
> ### Have you tried this code
> https://github.com/admob-plus/admob-plus/blob/master/examples/cordova-consent/www/js/index.js
> 
> ### If the application has been released
> https://support.google.com/fundingchoices/answer/9180084

I did, but that's a cordova example, I need an ionic example.

The ionic example on that project does not have Consent.

**Comment by ratson** [comment]
@felipe-ff No official Ionic plugin for `cordova-plugin-consent`, no plan to add that unless someone is funding it.
Or you could build one, then I could reference it in the docs.

**Comment by pirpy76** [comment]
Is it possible to use cordova-plugin-consent in a ionic project? If yes, how? Can someone make an example?
